docs/Project-Configuration/Containerized-Tasks.md

```diff
@@ -0,0 +1,118 @@
+# Containerized Tasks
+
+We're excited to introduce the ability to run Evergreen tasks in containers. 
+
+This offering is designed to streamline work and reduce friction caused by software dependency requirements. 
+Greater flexibility and control over task environments is achievable with containers, ensuring that each task runs in an isolated, dedicated space with its own specific set of dependencies. 
+
+This means fewer conflicts, 
+more consistent results, and the potential for more efficient execution as multiple tasks can run simultaneously on the same host, enabling more efficient utilization of resources, and reducing the likelihood of scheduling delays due to unavailable hosts.
+
+## Important Note
+Container tasks are still undergoing active testing and, at this stage, is used by the internal Evergreen team only.
+
+If you have any questions about container tasks or are interested in exploring how this feature could benefit your project, we encourage you to reach out to us. 
+We'll discuss its potential applications and assist you in preparing for its broader release.
+
+## What's Different
+
+While tasks running on containers come with all the capabilities you're familiar with in regular host tasks, there are a couple of differences to note as we roll out this new feature:
```

Comment:
To make it extra clear, we should also mention that containers are _not_ the same as hosts, which may require some user adjustment. For example, things that are heavily reliant on low-level components (such as systemd or Docker) are not available within a container. Might also be worth linking some kind of Docker docs or something about the differences (if one exists, I imagine it does).

Reply:
Closest thing I could find was the [docker architecture overview](https://docs.docker.com/get-started/overview/#docker-architecture).